### DISCUSSION TITLE
R session aborted when load Seurat

### DISCUSSION THREAD

**Discussion by LIPEISHAO** [satijalab/seurat#6933] [Q&A]
Hi All,

I encountered one problem these days. R session always aborted when I load Seurat package.
Here is some information:
> sessionInfo()
R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19044)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] BiocManager_1.30.19 compiler_4.1.0      tools_4.1.0         data.table_1.14.6 

Any suggestions will be helpful. Thanks in advance.

**Comment by Dragonmasterx87** [comment]
Do you use a IDE like Rstudio? I would suggest Uninstalling your libraries and R and re-installing it. This sounds like a problem in your base R installation that has become corrupt and it is not productive to identify the problem, a fresh installation is more productive in this case.

**Reply by sevves7** [reply]
I tried your suggestion by using the R rather than R-studio to apply this package, and it cannot work as well. Does this suggest that there is a possibility that the base R installation is wrong?  I update my R using the code😭
```
library(installr)
updateR()
```
my sessioninfo() is
```
R version 4.2.2 (2022-10-31 ucrt)
```

**Reply by Dragonmasterx87** [reply]
Rstudio doesn't use an independent version of R it uses an available version of R installed on your PC. What I was suggesting was, first uninstalling R, its libraries, and Rstudio, and then performing a system restart fresh installation and a final system restart. This should resolve your problem.

You can try this, but I dont think it will work because most likely there is a bug in the dependency of seurat not seurat itself:

```
options(seurat.debug = TRUE)
library(seurat)
```

Finally, you can also try the following: 1. uninstall Rstools and install rtools v4.2 and then perform the following:

```
install.packages("Seurat")
databases_backup <- available.packages()
databases_backup_initialize <- tools::package_dependencies("Seurat", databases_backup)$Seurat
install.packages(databases_backup_initialize)
library(Seurat)
```

**Reply by sevves7** [reply]
Thank you! Installing Rstools v4.2 and performing the codes successfully solve my problem! I have wasted for more than a week on this problem! Thank you very much!!❤

**Reply by Dragonmasterx87** [reply]
No problem! Glad it worked.
